.github/workflows/ci.yml

```diff
@@ -1,165 +0,0 @@
-
-      - name: Build and Package
-        run: mvn -B dependency:go-offline clean package -DskipTests=true
-
-      - name: Save Artifacts
-        uses: actions/upload-artifact@v2
-        with:
-          name: liquibase-hibernate-artifacts
-          path: |
-            target/*.jar
-
-      - name: Save Event File
-        uses: actions/upload-artifact@v3
-        with:
-          name: Event File
-          path: ${{ github.event_path }}
-
-      - name: Build Cache
-        uses: actions/cache@v3.0.5
-        with:
-          key: build-${{ github.run_number }}-${{ github.run_attempt }}
-          path: |
-            **/target/**
-            ~/.m2/repository/org/liquibase/
-
-  unit-test:
-    name: Test Java ${{ matrix.java }}
-    runs-on: ubuntu-latest
-    needs: build
-
-    strategy:
-      fail-fast: false
-      matrix:
-        java: [ 17, 19 ]
-
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: Set up JDK ${{ matrix.java }}
-        uses: actions/setup-java@v2
-        with:
-          java-version: ${{ matrix.java }}
-          distribution: 'temurin'
-          cache: 'maven'
-
-      - name: Build Cache
-        uses: actions/cache@v3.0.5
-        with:
-          key: build-${{ github.run_number }}-${{ github.run_attempt }}
-          path: |
-            **/target/**
-            ~/.m2/repository/org/liquibase/
-
-      - name: Run Tests
-        run: mvn -B jacoco:prepare-agent surefire:test
-
-      - name: Archive Test Results
-        if: ${{ always() }}
-        uses: actions/upload-artifact@v2
-        with:
-          name: test-reports-jdk-${{ matrix.java }}
-          path: |
-            **/target/surefire-reports
-            **/target/jacoco.exec
-
-  hibernate-test:
-    name: Test Hibernate ${{ matrix.hibernate }}
-    needs: build
-    runs-on: ubuntu-latest
-
-    strategy:
-      fail-fast: false
-      matrix:
-        hibernate: [ "6.0.2.Final", "6.1.7.Final" ]
-
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: Set up JDK
-        uses: actions/setup-java@v2
-        with:
-          java-version: 17
-          distribution: 'temurin'
-          cache: 'maven'
-
-      - name: Build Cache
-        uses: actions/cache@v3.0.5
-        with:
-          key: build-${{ github.run_number }}-${{ github.run_attempt }}
-          path: |
-            **/target/**
-            ~/.m2/repository/org/liquibase/
-
-      - name: Run Compatibility Tests
-        run: mvn -B jacoco:prepare-agent surefire:test -Dhibernate.version=${{ matrix.hibernate }}
-
-      - name: Run Tests
-        run: mvn -B jacoco:prepare-agent surefire:test
-
-      - name: Archive Test Results
-        if: ${{ always() }}
-        uses: actions/upload-artifact@v2
-        with:
-          name: test-reports-hibernate-${{ matrix.hibernate }}
-          path: |
-            **/target/surefire-reports
-            **/target/jacoco.exec
-
-  dependabot:
```

Comment:
Seems we forgot to add the dependabot auto merge that was on ci.yml ? Or maybe I'm jumping the gun and it will be added to a another ci file?

Reply:
@filipelautert That's not something we have on our current reusable workflows.  Is that something we want on all extensions?

Reply:
Yes - I added it to all extensions before 4.23.0 as per https://datical.atlassian.net/browse/DAT-14777

Reply:
@filipelautert I copied the dependabot automerge feature to a reusable workflow and added the call to the bottom of this workflow.  I also added back the integration test steps.